### DISCUSSION TITLE
ForecastPFN for time series forecasting

### DISCUSSION THREAD

**Discussion by mglowacki100** [autogluon/autogluon#3792] [Ideas]
Hi @shchur, @Innixma,

I've noticed that TabPFN is included in AutoGluon as optional model. I'm curious if you've come across ForecastPFN. In a nutshell, it's the PFN idea adapted for time series (you can find it at https://github.com/abacusai/ForecastPFN and the research paper at https://arxiv.org/abs/2207.01848).

I was wondering if there are any plans to include ForecastPFN in AutoGluon. It comes with an Apache 2.0 license, but unfortunately, there isn't a PyPI package available. Despite sharing similar limitations with TabPFN, its focus on time series could potentially be advantageous for tackling cold start forecasting problems.

**Comment by shchur** [comment]
Thanks for the suggestion, @mglowacki100. We are looking into the feasibility of including pre-trained forecasting models in AutoGluon, but I don't yet have any specific plans / timelines to share.
